pandas/tests/frame/test_join.py

```diff
@@ -24,6 +24,14 @@ def right():
     return DataFrame({"b": [300, 100, 200]}, index=[3, 1, 2])
 
 
+@pytest.fixture(params=[True, False, None])
```

Comment:
Perhaps just define this as a `pytest.mark.parametrize` on the test itself? Since we only use it once that's a bit clearer.